src/MUIDataTable.js

```diff
@@ -2,6 +2,7 @@ import Paper from '@material-ui/core/Paper';
 import { withStyles } from '@material-ui/core/styles';
 import MuiTable from '@material-ui/core/Table';
 import classnames from 'classnames';
+import assign from 'lodash.assign';
```

Comment:
We can eliminate this; as of latest master, we're not using `assign` anymore (it's `assignwith`), but also doesn't appear needed given the change sets here.

Reply:
Removed.